code/modules/crew_objectives/medical_objectives.dm

```diff
@@ -13,6 +13,19 @@
 			return FALSE
 	return TRUE
 
+/datum/objective/crew/emtmorgue
+	explanation_text = "Ensure that no corpses remain outside of Medbay when the shift ends."
+	jobs = "paramedic,"
+
+/datum/objective/crew/emtmorgue/check_completion()
+	var/list/medical_areas_morgue = typecacheof(list(/area/medical/cryo, /area/medical/genetics/cloning, /area/medical/exam_room,
+		/area/medical/medbay/aft, /area/medical/medbay/central, /area/medical/medbay/lobby, /area/medical/patients_rooms,
+		/area/medical/sleeper, /area/medical/storage, /area/medical/morgue))
+	for(var/mob/living/carbon/human/H in GLOB.mob_living_list)
+		if(H.stat == DEAD && is_type_in_typecache(H, medical_areas_morgue)) // If person is dead and corpse is NOT in one of these areas
```

Comment:
need to know how to set it so that it gets marked as completed when there are no corpses that are NOT in medbay